klarna-payments-for-woocommerce.php

```diff
@@ -228,6 +262,23 @@ public function init() {
 			);
 			$this->siwk                    = new SignInWithKlarna( $settings );
 			$this->interoperability_token  = new KP_Interoperability_Token();
+			$this->logger                  = new Logger( 'klarna_payments', wc_string_to_bool( $settings['logging'] ?? false ) );
+
+			$report_about = array(
+				array(
+					'type'       => 'kp_section_start',
+					'is_section' => true,
+				),
+				array( 'type' => 'select' ),
+				array(
+					'type'    => 'checkbox',
+					'exclude' => array(
+						'empty' => 'title',
+					),
+				),
+
+			);
```

Comment:
Suggestion: I would suggest adding a comment here to describe what the different arrays do. Just to make it easier to maintain going forward as well.

Reply:
Question: Also the variable is called `$report_about`, is that a good description of what the array is here?

Reply:
How about `$included_settings_fields`?

Reply:
Is that what it is though? Since we have three arrays, one with `kp_section_start`, another with `select` and a third with `checkbox`, with a exclude. Does that mean all other fields are excluded from this feature? I know they are not, but in a year when we need to change this, it might be harder to tell at a glance.

Maybe we could have a quick talk about this internally and see if we can make it a bit easier to understand at a glance?